Makefile

```diff
@@ -112,3 +112,9 @@ poetry-dependencies:
 	# NOTE: pip does not yet support hash verification for git dependencies;
 	# rocky's requirements-dev.txt unfortunately has no hashing until then
 	sed -i '/--hash/d; s/ \\$$//' rocky/requirements-dev.txt
+
+
+local-ci-robot:
```

Comment:
Isn't this supposed to work?
```shell
$ cd rocky
$ make test-rf
```

It is not working for me I see so I suppose it hasn't been updated. I'd say we should remove that target (and `test-prepare`) and perhaps call this new make target in the github workflow?

Reply:
I have removed it till we find a better solution in another PR